src/lake/tests/cachedirtag/test.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Test that CACHEDIR.TAG file is correctly created in .lake directory.
+#
+# Removing CACHEDIR.TAG without removing .lake directory and then rebuilding
+# does not currently recreate CACHEDIR.TAG. This is an artifact of the current
+# implementation and is technically against the spec:
+#
+# > The application should also regenerate the cache directory tag if it
+# > disappears --
```

Comment:
This is an odd spec requirement that does not seem desirable. A user may wish to remove the tag and cache the directory and Lake should not override that.